src/org/zaproxy/zap/utils/Stats.java

```diff
@@ -26,128 +26,179 @@
 import java.util.Map;
 import java.util.Map.Entry;
 
+import org.apache.log4j.Logger;
+
 public class Stats {
```

Comment:
Should be final? And have a private default constructor? (all access are done statically)